provider/win_driver_utils.py

```diff
@@ -194,6 +194,71 @@ def copy_file_to_samepath(session, test, params):
                        " the detailed info:\n%s." % output)
 
 
+def enable_driver(vm, test, params, device_id):
+    """
+    Enable driver.
+
+    :param vm: vm object
+    :param test: kvm test object
+    :param params: the dict used for parameters
+    :param device_id: Device id in windows guest
+    """
+    session = vm.wait_for_login()
+    cmd = params["driver_enable_cmd"]
+    cmd = cmd.replace("DEVICE_ID", device_id)
+    cmd = utils_misc.set_winutils_letter(session, cmd)
+    status, output = session.cmd_status_output(cmd)
+    if status != 0:
+        test.fail("failed to enable driver, %s" % output)
+
+
+def disable_driver(vm, test, params, device_id):
+    """
+    Disable driver.
+
+    :param vm: vm object
+    :param test: kvm test object
+    :param params: the dict used for parameters
+    :param device_id: Device id in windows guest
+    """
+    session = vm.wait_for_login()
+    cmd = params["driver_disable_cmd"]
+    cmd = cmd.replace("DEVICE_ID", device_id)
+    cmd = utils_misc.set_winutils_letter(session, cmd)
+    status, output = session.cmd_status_output(cmd)
+    if status != 0:
+        if "reboot" in output:
+            vm.reboot()
+        else:
+            test.fail("failed to disable driver, %s" % output)
+
+
+def get_device_id(vm, test, params):
+    """
+    Get device id from guest.
+
+    :param vm: vm object
+    :param test: kvm test object
+    :param params: the dict used for parameters
+    """
+    device_name = params["device_name"]
+    device_hwid = params["device_hwid"]
```

Comment:
We have a function def get_hwids()  in provider/win_dev.py, maybe we can use it to get it automatically~